metadata.json

```diff
@@ -5,14 +5,25 @@
   "summary": "A helper for writing Bolt plugins in Ruby",
   "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-ruby_plugin_helper",
-  "dependencies": [],
+  "dependencies": [
+
+  ],
+  "operatingsystem_support": [
+    "RedHat based Linux",
+    "Debian based Linux",
+    "Windows"
+  ],
   "requirements": [
     {
       "name": "bolt",
       "version_requirement": "< 4.0"
+    },
+    {
+      "name": "puppet",
+      "version_requirement": ">= 6.0.0 < 9.0.0"
```

Comment:
```suggestion
      "version_requirement": ">= 7.0.0 < 9.0.0"
```

Reply:
there was no previous requirement so I wanted to give a bit more leeway here, but I guess it make more sense to only list what has been tested